### DISCUSSION TITLE
Immune repertoire profiling with bulk RNASeq

### DISCUSSION THREAD

**Discussion by dbottomly** [milaboratory/mixcr#998] [General]
Hi:

I am working on a feasibility study using bulk RNASeq and am considering the use of MiXCR.  Your documentation is great!  However I came across a related couple questions I am hoping you can answer:

1. In one of the MiXCR [papers](https://www.nature.com/articles/nbt.3979#Sec1) (Fig S4 specifically), the Ig isotype proportions as well as related stats such as the IgG1/IGH proportion are discussed.  What is the best way to compute these values in MiXCR?
2. Another [paper](https://ascopubs.org/doi/suppl/10.1200/JCO.21.01506/suppl_file/DS1_JCO.21.01506.pdf) computed diversity measures with respect to specific Ig classes (e.g. IGHG).  However, it looks like `postanalysis` would only produce diversity measures for IGH (or other specified chains)?

Thanks,

Dan

**Comment by PoslavskySV** [comment]
Hi Dan,

thank you for reaching us. Regarding your questions:

1. Fig. 4 is quite advanced and the default postanalysis in MiXCR currently not suited for building such complex plots. So my advice is to export clones with all columns you need and calculate all values you need manually. The 'Best C Hit' in the output should be used to pick up the isotype.
2. For now you are right, the current postanalysis will not split the data by isotype.

These are the limitations of current postanalysis functionality. But we are working on this and in Q2 we are going to launch a much more powerful and flexible downstream analysis capabilities, so stay tuned )

BR,
Stan
